scripts/utils/check-locales.ts

```diff
@@ -0,0 +1,168 @@
+import fs from 'fs';
+import path, { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import lodash from 'lodash';
+import { logger } from '../utils/logger';
+
+const { flatten, fromPairs, uniq } = lodash;
+
+const currentFilePath = fileURLToPath(import.meta.url);
+const currentDirPath = dirname(currentFilePath);
+
+const localesDir = path.resolve(
+  currentDirPath,
+  '..',
+  '..',
+  'public',
+  'locales',
+);
```

Comment:
```suggestion
const localesDir = path.resolve(currentDirPath, '..', '..', 'public', 'locales');
```

Reply:
why? that's a prettier thing?

Reply:
done by prettier actually, will throw an error if not done that way